lib/Core/Site/Values/Content.php

```diff
@@ -388,11 +395,17 @@ private function getOwner()
             return $this->owner;
         }
 
-        try {
-            $this->owner = $this->site->getLoadService()->loadContent($this->getContentInfo()->ownerId);
-        } catch (NotFoundException $e) {
-            // Do nothing
-        }
+        $loadService = $this->site->getLoadService();
```

Comment:
You don't have to do this. Closures are automatically bound to the object, so they have access to `$this`.